central/administration/events/datastore/internal/writer/writer.go

```diff
@@ -11,17 +11,16 @@ import (
 // Writer implements a buffered write for the administration events datastore.
 //
 // Since we generate events from logs, there could be many log events created
-// in a small time frame. The buffer is essentially a write optimization such
+// in a small-time frame. The buffer is essentially a write optimization such
```

Comment:
Why this change?

Reply:
Goland's spell check 🤷

Reply:
Yeah it's cause it should have been `small timeframe` from the start, so changed it to that.